_includes/configuration/toolbar.md

```diff
@@ -48,6 +48,25 @@ tinymce.init({
 });
 ```
 
+### Enabling Sticky Toolbars
+
+A Sticky Toolbar (or Docking Toolbar), docks the toolbar to the top of the screen when scrolling down a web page until the editor is no longer visible. To enable Stick toolbars declare `toolbar_sticky` as `true` in the `tinymce.init` script. To see a demo of sticky toolbar, see the [Full featured demo]({{ site.baseurl }}/demo/full-featured/).
+
+**Type:** `Boolean`
+
+**Default Value:** `false`
```

Comment:
Should this be `true` based on what @EkimChau said on Friday?

Reply:
True that :)

Reply:
Updated